src/main/java/org/opensearch/security/user/UserService.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2015-2018 _floragunn_ GmbH
```

Comment:
This doesn't seem correct

Reply:
It certainly is not haha. Copied from the wrong file it appears. Good catch.